src/worldofzuul/Room.java

```diff
@@ -11,22 +11,48 @@
  */
 public class Room {
 
+    /**
+     * The String "description" is used when describing the room generated with the constructor
+     * The HashMap is a list of existing rooms
+     */
     private String description;
     private HashMap<String, Room> exits;
```

Comment:
HashMap delen skal flyttes ned over denne attribute.

Reply:
Fixed in 92bf61c54f269f9d29829785be5f57a245188487